powersimdata/input/input_data.py

```diff
@@ -56,10 +56,11 @@ def get_data(self, scenario_info, field_name):
         ext = self.file_extension[field_name]
 
         if field_name in ["demand", "hydro", "solar", "wind"]:
-            interconnect = interconnect2name(scenario_info["interconnect"].split("_"))
             version = scenario_info["base_" + field_name]
-            file_name = interconnect + "_" + field_name + "_" + version + "." + ext
-            from_dir = server_setup.BASE_PROFILE_DIR
+            file_name = field_name + "_" + version + "." + ext
+            from_dir = posixpath.join(
+                server_setup.BASE_PROFILE_DIR, scenario_info["grid_model"]
+            )
```

Comment:
If someone ever tries to run the 'server' on a Windows machine, this could cause trouble, but I think that will be rare.

Reply:
@jon-hagg, did you create a class that creates path smartly. I remember something in that vein

Reply:
I did something, but not smart enough for this case. Anyway, don't think it's an issue - either we will host the server, or users can run the linux container, which still works on windows.